front-end/src/components/SectionCard/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+import styles from './SectionCard.module.css'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faEdit } from '@fortawesome/free-regular-svg-icons'
+export const SectionCard = () => {
+  return (
+    <div className={styles.container}>
+      <h1 className={styles.title}>
+        This is a Title
+        <p className={styles.username}>User Name</p>
+      </h1>
+
+      <FontAwesomeIcon
+        onClick={() => console.log('Clicked Edit button')}
```

Comment:
There is an `Icon` component by the way, maybe it would be nice to use that one instead and add the `onClick` property to that component as well? The rest of the code looks good to me :)